[2025-02-06T20:34:43.450+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-06T20:34:43.460+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.featur_engineer scheduled__2025-02-03T13:30:00+00:00 [queued]>
[2025-02-06T20:34:43.462+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_hooks.featur_engineer scheduled__2025-02-03T13:30:00+00:00 [queued]>
[2025-02-06T20:34:43.463+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-06T20:34:43.467+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): featur_engineer> on 2025-02-03 13:30:00+00:00
[2025-02-06T20:34:43.474+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=222) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-06T20:34:43.475+0000] {standard_task_runner.py:64} INFO - Started process 249 to run task
[2025-02-06T20:34:43.480+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_hooks', 'featur_engineer', 'scheduled__2025-02-03T13:30:00+00:00', '--job-id', '1536', '--raw', '--subdir', 'DAGS_FOLDER/Stock_pred.py', '--cfg-path', '/tmp/tmpmtpykv7y']
[2025-02-06T20:34:43.481+0000] {standard_task_runner.py:91} INFO - Job 1536: Subtask featur_engineer
[2025-02-06T20:34:43.501+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_postgres_hooks.featur_engineer scheduled__2025-02-03T13:30:00+00:00 [running]> on host b7aba691e1ff
[2025-02-06T20:34:43.541+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_postgres_hooks' AIRFLOW_CTX_TASK_ID='featur_engineer' AIRFLOW_CTX_EXECUTION_DATE='2025-02-03T13:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-03T13:30:00+00:00'
[2025-02-06T20:34:43.541+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-06T20:34:43.954+0000] {warnings.py:112} WARNING - /opt/***/dags/Stock_pred.py:237: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  input.drop(columns=["Target","CLOSE_t+1"],inplace=True)

[2025-02-06T20:34:43.963+0000] {warnings.py:112} WARNING - /opt/***/dags/Stock_pred.py:240: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  input['CLOSE_diff1'] = input['CLOSE'].diff(1)

[2025-02-06T20:34:43.972+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-06T20:34:43.972+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-06T20:34:43.976+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_hooks, task_id=featur_engineer, run_id=scheduled__2025-02-03T13:30:00+00:00, execution_date=20250203T133000, start_date=20250206T203443, end_date=20250206T203443
[2025-02-06T20:34:43.994+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-02-06T20:34:44.012+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-06T20:34:44.012+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
